uPortal-webapp/src/main/webapp/WEB-INF/jsp/Search/searchRest.jsp

```diff
@@ -0,0 +1,222 @@
+        <span class="up-search-list-item-secondary-content pull-right">
+            <a class="up-search-list-item-secondary-action" href="#">
+                <i class="fa fa-window-maximize" aria-hidden="true"></i>
+            </a>
+        </span>
+        <span class="up-search-list-item-primary-content">
+            <i class="up-search-list-item-avatar pull-left fa"></i>
+            <h3 class="up-search-list-item-title list-group-item-heading"></h3>
+            <span class="up-search-list-item-body list-group-item-text">
+                <dl></dl>
+            </span>
+        </span>
+    </li>
+</template>
+
+<template id="search-result-item-detail-template">
+    <dt></dt>
+    <dd></dd>
+</template>
+
+<template id="search-result-no-results-template">
+    <div class="h3 text-center">
+         <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+         No Results Found
+         <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+     </div>
+</template>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.4/lodash.min.js"></script>
+<!-- Does the above replace underscore? -->
+<script src="https://rawgit.com/github/fetch/v2.0.3/fetch.js"></script>
+<script src="https://rawgit.com/webcomponents/template/v1.0.0/template.js"></script>
+<script src="https://rawgit.com/taylorhakes/promise-polyfill/6.0.2/promise.js"></script>
+<!-- WTF are the three above? -->
+<style>
+    #search-results-tab-header {
+        display: inline-block;
+    }
+
+    .up-search-list {
+        margin: 0 6rem;
+    }
+
+    .up-search-list .up-search-list-item .up-search-list-item-avatar {
+        margin-left: 2rem;
+        font-size: 4rem;
+    }
+
+    .up-search-list .up-search-list-item .up-search-list-item-title {
+        font-weight: bold;
+    }
+
+    .up-search-list .up-search-list-item .up-search-list-item-title,
+    .up-search-list .up-search-list-item .up-search-list-item-body dl {
+        margin-left: 8rem;
+        margin-bottom: .5rem;
+    }
+
+    #search-results-tab-panel dt,
+    #search-results-tab-panel dd {
+        display: inline;
+    }
+
+    #search-results-tab-panel dt {
+        font-weight: bold;
+    }
+
+    #search-results-tab-panel dt::before {
+        content: '';
+        display: block;
+    }
+
+    .up-search-list .up-search-list-item .up-search-list-item-secondary-content {
+        margin-right: 2rem;
+    }
+
+    .up-search-list .up-search-list-item .up-search-list-item-secondary-action i {
+        font-size: 2rem;
+    }
+</style>
+<div id="search-results-tab-panel" class="">
+    <div class="text-center">
+        <ul id="search-results-tab-header" class="nav nav-pills"></ul>
+    </div>
+    <div id="search-results-tab-content" class="tab-content clearfix"></div>
+</div>
+
+<script language="javascript" type="text/javascript">
+// search results metadata
+// TODO: This solution doesn't honor local configuration or internationalization
```

Comment:
:question: can this be resolved before these changes are merged?

Reply:
Yes, done.  That was a PITA.